apps/framework-cli/src/cli/routines.rs

```diff
@@ -297,7 +297,7 @@ pub async fn start_production_mode(project: Arc<Project>) -> anyhow::Result<()>
     let mut route_table = HashMap::<PathBuf, RouteMeta>::new();
 
     info!("<DCM> Initializing project state");
-    let (framework_object_versions, version_syncs) =
+    let (framework_object_versions, versions_syncs) =
```

Comment:
The extra plural "s" sounds very awkward to me.

Reply:
I don't have a strong feeling, will change back